### PR TITLE
add gnosis chain and the new experimental ywxdai vault

### PR DESCRIPTION
**Pull request by shegenerates**

### CODE REVIEW

File: utils/chains.json [hunk 1]
```diff
@@ -77,5 +77,12 @@
         "decimals": 18
       }
     }
+  },
```
Comment (saltyfacu) [inline utils/chains.json:80]: to add a new chain, you need to also add all inside `chain_swap`

Example below:

```
    "chain_swap": {
      "chainId": "0xA4B1",
      "blockExplorerUrls": ["https://arbiscan.io"],
      "chainName": "Arbitrum One",
      "rpcUrls": ["https://arb1.arbitrum.io/rpc"],
      "nativeCurrency": {
        "name": "AETH",
        "symbol": "AETH",
        "decimals": 18
      }
     

```

Comment (shegenerates) [reply utils/chains.json:80]: added!

File: utils/chains.json [hunk 2]
```diff
@@ -52,7 +66,12 @@
       "chainId": "0x89",
       "blockExplorerUrls": ["https://polygonscan.com"],
       "chainName": "Matic(Polygon) Mainnet",
-      "rpcUrls": ["https://rpc-mainnet.matic.network", "wss://ws-mainnet.matic.network", "https://rpc-mainnet.matic.quiknode.pro", "https://matic-mainnet.chainstacklabs.com"],
+      "rpcUrls": [
+        "https://rpc-mainnet.matic.network",
+        "wss://ws-mainnet.matic.network",
+        "https://rpc-mainnet.matic.quiknode.pro",
+        "https://matic-mainnet.chainstacklabs.com"
+      ],
```
Comment (shegenerates) [inline utils/chains.json:74]: this is all just linting

File: utils/vaults.json [hunk 3]
```diff
@@ -1019,5 +1019,17 @@
...
+    "ywxdai": {
+    "TITLE": "Basic Wrapped xDai Farmer",
+    "LOGO": "🏆🚀",
+    "VAULT_ABI": "yVaultV2",
+    "VAULT_TYPE": "experimental",
+    "VAULT_ADDR": "0xE35f0516D330395a3878eB60114349b6325ec02E",
+    "WANT_ADDR": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    "WANT_SYMBOL": "wxDai",
+    "COINGECKO_SYMBOL": "wrapped-xdai",
+    "VAULT_STATUS": "withdraw",
```
Comment (saltyfacu) [inline utils/vaults.json:1032]: the status should be "new". If now it won't show it unless you have funds in it. There's a script that will create this for you :)

**Comment by shegenerates** [comment]
Success! 
Screen Shot 2022-02-07 at 2 06 44 PM

**Comment by poolpitako** [comment]
You can also add yvUSDC if you want: 0x21d7B09Bcf08F7b6b872BED56cB32416AE70bCC8
Tested and it works now. LGTM.

**Review by poolpitako — APPROVED** [review]

File: scripts/newVault.js [hunk 4]
```diff
@@ -107,12 +110,12 @@ if (!args.logo) {
 **	The chain will be associated with the `CHAINID` key. Numbers are expected.
 **	Possible value : 1, 56, 137, 250
```
Comment (saltyfacu) [inline scripts/newVault.js:111]: it'd be nice to have the network id in this comment too

**Review by saltyfacu — APPROVED** [review]
really good job! looking good. jus left a tiny comment

**Comment by shegenerates** [comment]
@saltyfacu could you merge it when you're ready then please? I cannot since I do not have write access. Or add me as a contributor! :)

**Pull request merged**